docs/literate/src/files/subcell_shock_capturing.jl

```diff
@@ -73,6 +73,7 @@
 # The procedure to enforce global bounds for a conservative variables is as follows:
 # If you want to guarantee non-negativity for the density of the compressible Euler equations,
 # you pass the specific quantity name of the conservative variable.
+using OrdinaryDiffEqLowStorageRK
```

Comment:
```suggestion
```
This should not be necessary anymore as this was only added for the `CallbackSet`.